package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "vui-alerts",
   "version": "0.1.4",
-  "description": "Mixins and CSS for styling alerts",
+  "description": "Web component alert elements",
+  "private": true,
   "scripts": {
-    "clean": "rimraf *.css && rimraf ./test/*.css",
-    "prebuild": "npm run clean",
-    "build:sass:test": "node-sass --output-style expanded ./test/test.scss ./test/test.css",
-    "build:sass": "node-sass --output-style expanded error.css.scss error.css",
-    "build": "npm run build:sass && npm run build:sass:test",
-    "perceptual-diff": "percy snapshot test/",
-    "pretest": "npm run build",
-    "test": "csslint error.css"
+    "prebuild": "cpy *.html ./bower_components/temp",
+    "build": "vulcanize ./bower_components/temp/alert.html > ./test/perceptual/alert.html.ignore",
+    "postbuild": "rimraf ./bower_components/temp",
+    "preperceptual-diff": "npm run build",
+    "perceptual-diff": "percy snapshot test/perceptual --enable_javascript",
+    "postinstall": "bower install",
+    "screenshot": "phantomjs ./screenshots/build.js",
+    "test": "polylint && wct --skip-plugin sauce"
```

Comment:
I don't think we're doing this consistently yet, but in addition to Polylint (which catches Polymer specific WC issues), I think we also want to run eslint on the JS.  Just something to be aware of that we'll likely start doing, in particular for components that have behavior implemented in JS (which this ones does not have yet)...  ;)

Also note: we're still in the process of getting Percy perceptual diff working for WCs.